casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
```

Comment:
My reasoning was that HostInfo calls with use_aiprs=False really want to rely on defaults, and only the ones with use_aipsrc=True really want something customized.